sensitivity.tex

```diff
@@ -131,11 +130,11 @@ \subsection{Lifetime Extension of French \glspl{LWR}}\label{sec:life}
 	\label{fig:ext_uox}
 \end{figure}
 
-Figure \ref{fig:ext_all} shows that lifetime extension has little
-effect on the \emph{average} monthly reprocessing demand, because
-the amount of plutonium in the \gls{ASTRID} used fuel remains the same.
+Figure \ref{fig:ext_all} shows that lifetime extension increases
+\emph{average} monthly reprocessing demand, because more \gls{LWR}
+\gls{UNF} is reprocessed with delayed \gls{ASTRID} deployment.
```

Comment:
again, a little confusing. Why must more LWR UNF be reprocessed if ASTRID deployment is later... ?